allauth/socialaccount/providers/oauth2/client.py

```diff
@@ -11,6 +11,8 @@ class OAuth2Error(Exception):
 
 
 class OAuth2Client(object):
+    CLIENT_ID_PARAMETER = "client_id"
```

Comment:
```suggestion
    client_id_parameter = "client_id"
```

The other constant properties of this class (such as e.g. `scope_delimiter` use just lower case.